pkg/controller/controller_instance.go

```diff
@@ -208,7 +208,8 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) error {
 				return fmt.Errorf("Couldn't create a key for object %+v: %v", instance, err)
 			}
 			c.pollingQueue.Add(key)
-		} else {
+		} else if respCode == http.StatusCreated || respCode == http.StatusOK {
```

Comment:
I'm a little perplexed that this is necessary because above L163 any error should be handled and these are the only cases that return a nil error so I'm just trying to understand why this change is necessary.